sdk/program/src/epoch_rewards.rs

```diff
@@ -7,6 +7,8 @@
 //! [`sysvar::epoch_rewards`]: crate::sysvar::epoch_rewards
 
 use std::ops::AddAssign;
+
+#[repr(C)]
 #[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Default, Clone, Copy, AbiExample)]
 pub struct EpochRewards {
```

Comment:
Just putting this here for completeness: this sysvar will get reworked as soon as https://github.com/solana-foundation/solana-improvement-documents/pull/118 lands, and `repr(C)` is part of the new spec.